lib/puppet/util/hdp.rb

```diff
@@ -31,56 +31,56 @@ def get_trusted_info(node)
 
   def submit_command_to_hdp(host, command, version, certname, producer_timestamp_utc, payload)
     checksum_payload = Puppet::Util::Puppetdb::CharEncoding.utf8_string({
-      :command => command,
-      :version => version,
-      :certname => certname,
-      :payload => payload,
+      command: command,
+      version: version,
+      certname: certname,
+      payload: payload,
     }.to_pson, "Error encoding a '#{command}' command for host '#{certname}'")
 
-    command = Puppet::Util::Puppetdb::CharEncoding.coerce_to_utf8(command).gsub(" ", "_")
```

Comment:
This was scraped right from PuppetDB Src - is this something that is just a warning/style issue, or should they be notified?

Reply:
looking, one sec

Reply:
ah, likely just a rubocop best practice